### DISCUSSION TITLE
Selaginella

### DISCUSSION THREAD

**Discussion by joelnitta** [pteridogroup/ppg#29] [General]
I am personally not in favor of splitting *Selaginella*, but posting this paper here in case folks want to discuss it:

Zhou X-M, Zhang L-B (2023) Phylogeny, character evolution, and classification of Selaginellaceae (lycophytes). Plant Diversity S2468265923001026. https://doi.org/10.1016/j.pld.2023.07.003

[PDF](https://www.dropbox.com/s/ynj32ydabsvwi7p/Zhou_Zhang_2023_Phylogeny%2C_character_evolution%2C_and_classification_of_Selaginellaceae.pdf?dl=0)

**Comment by DrAshleyField** [comment]
Do we (PPG) have a policy regarding proposals and voting on classifications that are dependent on current Nomenclature proposals?

The multi-genus Selaginellaceae classification by Zhou and Zhang (2023) is underpinned by a type change for the genus (see Wan et al. 2023 [https://doi.org/10.1002/tax.12918](https://doi.org/10.1002/tax.12918)).

Could this be a curious circular situation where the decision of the nomenclature committee has a bearing on our decision as PPG and/or visa versa?

Ash

**Reply by joelnitta** [reply]
Good point @DrAshleyField! I was not aware of that proposal. We do not currently have a policy about this (it had not come up before, nor did we anticipate it). What do others think? (@crothfels, @schuettpelze)

**Reply by schuettpelze** [reply]
In the guidelines, we stipulate that: "Any higher-level taxon (genus and above) proposed for recognition needs to be published (taxa “in press”, etc., are not allowed). Similarly, all supporting studies should be published." Conservation proposals are different in that publication itself is not the end of the road. I suppose we could require the process run to completion (post IBC) or we could choose some point along the way were it is reasonably certain that a proposal will ultimately be approved? As precedent, a proposal to conserve the name Drynaria against Aglaomorpha was still pending while PPG1 was being written and Aglaomorpha was thus used. In this case, the proposal was ultimately accepted and the name in PPG1 is "wrong".

**Reply by GuguYanez** [reply]
Hi! I think we should only accept the proposals that were actually approved at the last IBC. Although we can weigh the potential "acceptability" in the future, this can become quite subjective, why accept some and not others? Likewise, if we agree to recognize the IBC as a legitimate area to decide whether or not to accept nomeclatural issues, then we should not attribute that power to ourselves. Perhaps we can consider, however, adding a "notes" section where it is discussed that there is a proposal that could potentially change the name that we accept in the article. What do you think?

**Reply by choess** [reply]
It's implicit, rather than explicit, but if we maintain the PPG I circumscription of _Dennstaedtia_ (less _Mucura_ and _Sitobolium_) and _Microlepia_, we're assuming the success of proposal 2892 ([https://onlinelibrary.wiley.com/doi/full/10.1002/tax.12756](https://onlinelibrary.wiley.com/doi/full/10.1002/tax.12756)), conserving _Dennstaedtia_ with a new type. Proposal 2942 ([https://onlinelibrary.wiley.com/doi/full/10.1002/tax.12917](https://onlinelibrary.wiley.com/doi/full/10.1002/tax.12917)) also affects _Mucura_.

I think I would incline to be more open and say that for any PPG II proposals that depend in part on conservation, the conservation proposal should be published and should be disclosed in the PPG II proposal. Whether a given proposal is likely to succeed or not is a subjective judgment, but there is also some subjectivity to how we judge the strength of morphological and phylogenetic evidence.

I don't know what evidence is admissible under the bylaws of the standing Committees on Nomenclature, but endorsement of conservation proposals by PPG II might be taken as suggesting that they are desirable to the community. Of course, there are sometimes surprises (e.g., the proposal to conserve _Drynaria_ over _Aglaomorpha_ was neither recommended nor rejected by the Committee for Vascular Plants, but was recommended by the General Committee). If a conservation proposal we accepted were rejected by the IBC or its committees, I would feel bound to go back and revise our recommendations in the next round of PPG, but if we don't consider conservation proposals in their early stages and those proposals are accepted, we would also have to go back and make revisions.

**Reply by GuguYanez** [reply]
Yes, you are absolutely right about that. So, would it be too weird to think of waiting to submit the PPG II publication until after the next IBC? That is, we can continue working with the premise of accepting the proposals that, as a community, we agree should be approved, but wait to send the final publication until after the committee meets. 

I understand that it is quite a long time, almost a year, but not as long as the time that will pass until the committee meets again in ¿2028?

Well, they are just ideas to try to make the final version of PPG II as stable as we can over time :)

**Reply by gonggashan88** [reply]
Hi Joel, just curious: do you prefer to recognize a single genus, Lycopodium, in Lycopodiaceae (380 spp.) too?

**Reply by DrAshleyField** [reply]
@joelnitta @schuettpelze @crothfels Is it possible for us to make a PPG policy proposal regarding Nomenclature proposals and for the PPG II community to vote on that policy? A policy proposal could be raised as a GitHub Issue the same as a PPG II classification proposal and voted on it using the same platform. Any prior classification proposals affected by the policy would need to be listed and then if needed revisited.

I feel that voting on PPG II proposals for which there remain unresolved Nomenclature proposals has issues, and for stability, I would prefer to hold off until after the Nomenclature decision has been made. If that is not available, we would essentially need two votes: a vote factoring the Nomenclature proposal being sustained, and a vote factoring the Nomenclature proposal being rejected.

**Reply by joelnitta** [reply]
@gonggashan88 No, but that is a separate matter. Taxonomy is based on both practicality and science. Just because it is possible to split a group into many genera (and even may make sense scientifically), it may not be practical to do so. There is a cost to making wide-ranging taxonomic changes that is borne by the users of that taxonomy. In the case of *Selaginella*, I am not convinced at the moment that the need to split outweighs the cost. (That is not to say I could not be convinced in the future; that is why we have discussions).

**Comment by Ivan-Valdespino** [comment]
As a matter of full disclosure, I should inform you that I strongly oppose splitting Selaginella as proposed by Zhou & Zhang (2023), and relevant arguments against the proposal by Wan et al. (2023) were already submitted for publication.

Regarding the matter of PPGII including or not proposals that could be affected by IBC decisions, it seems appropriate to: a) discuss them and get an idea as to what interested parties have to say about them but not to make any formal recommendations as to approval or disapproval for inclusion in PPG II; b) wait to make a formal decision to include or not them in PPG II based on discussions in this forum and on final decisions by the IBC. I believe these positions cover what Choess and GuguYanez have already said. It also allows for applying lessons learned based on the example mentioned by Schuettpelze regarding the proposal to conserve Drynaria against Aglaomorpha. 

Furthermore, I don't think that discussions in this forum should be submitted to the IBC in order to influence, in one way or another, final decisions by that body, unless a full report containing all points of view expressed here is submitted to them.

Finally and back to Selaginella, the nomenclatural issue, among others, is far from being resolved vis-à-vis Wan et al. (2023) and Zhou & Zhang (2023) since the latter recognized, for example, the genus name Didiclis. However, Didiclis was proposed for rejection in favor of Selaginella by Mazundar et al. (2018. Taxon 67: 442). This proposal was accepted by the Nomenclature Committee for Vascular Plants (Applequist, 2022. Taxon 71: 219-227) and the General Committee (Wilson, 2023. Taxon 72: 422-424).

**Reply by gonggashan88** [reply]
Hi Ivan, just a clarification about the Code. You can restore any rejected name as long as it does NOT include the type of the conserved name. 
In this case you mentioned, Selaginella and Didiclis have different types.  NO problem here at all.

**Comment by gonggashan88** [comment]
No matter whether the IBC will approve our proposal (Wan et al. 2023) to conserve Selaginella with a conserved type, S. flabellata (resolved in the former S. subg. Stachygynandrum), you can adopt or reject the Zhou & Zhang (2023) classification. The only difference is that if you adopt this new classification and the IBC disapproves the proposal, then the ca. 230 species of Selaginella s.s. from the New World all need to transfer to Stachygynandrum (ca. 225 new combinations are needed), which would not be a good idea.

**Comment by gonggashan88** [comment]
I understand that it will be hard to have the majority of the votes to adopt our Zhou & Zhang (2023) classification of Selaginella. It is a dramatical deviation anyway from the current one-genus classification. However, future generations might find it informative and practical to recognize these deeply diverged (85-370 million years ago) lineages with different morphologies and distributions. The nomenclatural benefit of our Taxon proposal is obvious, if the new classification will be adopted (or in future); if the new classification will never be adopted, our Taxon proposal to choose a different type for Selaginella would do no harm at all to botany.

PPG I (& II) adopted/will adopt 16-17 genera in Lycopodiaceae (380 spp.), 25 genera in Blechnaceae (ca. 265 spp.), 37 genera
in Thelypteridaceae (ca. 1200 spp.), and 33-42 genera in Polypodiaceae subf. Grammmitidoideae (ca. 900 spp.), why NOT 19 genera in Selaginellaceae (ca. 800 spp.) with diagnosable lineages that are much older than most genera in ferns and lycophytes?

**Comment by gonggashan88** [comment]
Thanks, Joel, for your response. Yes, it takes time. It reminds me of the
segregates of Lycopodiaceae. In 1960s, 1980s when J. Holub described
Diphasiastrum, Lateristachys, Lycopodiella, Lycopodiastrum, Phlegmariurus,
Pseudodiphasium, Pseudolycopodiella, Pseudolycopodium, etc., few people
recognized them and often all the 350+ species were treated in Lycopodium
s.l. (and Huperzia) until 2014, >30 years later. PPG I recognized 16
genera, quite a bit of change!
---
Many thanks and best wishes, Libing
Li-Bing Zhang, Dr. nat., Curator, Missouri Botanical Garden
www.mobot.org/Li-Bing Zhang
<https://www.missouribotanicalgarden.org/plant-science/plant-science/research-staff/article/2628/zhang-li-bing>

**Comment by gonggashan88** [comment]
I understand that the timing of the IBC nomenclature meeting is a little
bit of concern. Regarding the Zhou & Zhang classification of
*Selaginella *(uncorrected
proof: https://doi.org/10.1016/j.pld.2023.07.003) and the Wan et al. 2023
Taxon proposal (onlinelibrary.wiley.com/doi/full/10.1002/tax.12918), there
are four scenarios:

   1. PPG II adopts the new classification + the IBC approves the Wan et
   al. 2023 Taxon proposal: the ca. 230 New World names of *Selaginella *retain
   (minimal name changes);
   2. PPG II adopts the new classification + the IBC disapproves of the Wan
   et al. 2023 Taxon proposal: the ca. 230 New World names of *Selaginella*
   must be transferred to *Stachygynandrum *(maximal name changes);
   3. PPG II rejects the new classification + the IBC approves the Wan et
   al. 2023 Taxon proposal: only the type of *Selaginella *will be changed
   to a different species, and nothing else; and
   4. PPG II rejects the new classification + the IBC disapproves of the
   Wan et al. 2023 Taxon proposal: nothing will be changed.

Conclusions:

   1. The IBC decision will affect the name of one genus (*Selaginella
   s.s. *and 230 species names in it) only; and
   2. It will be in the best interest of our community to achieve the
   maximal name stability if the IBC approves the Wan et al. 2023 Taxon
   proposal.

PPG II might not adopt the new classification, but what about PPG III or
PPG IV? Next-generation scholars of *Selaginella* might recognize the
deeply diverged (up to 370 million yrs ago; FYI, all 24 polypods families
diverged after 175 mya, Du et al., 2021), morphologically diagnosable, and
geographically coherent lineages as genera. Also, up to now, at least 13
historical and recent (e.g., Weakley, 2012, 2022) taxonomic works have
already split *Selaginella s.l. *into several genera, no matter whether PPG
I or II adopted or will adopt them or not. I expect more and more people to
adopt them in the coming years/decades.

I don't think Selaginellales/Selaginellaceae/*Selaginella s.l. *(750+ spp.)
will be retained in one single genus forever.

---
Best wishes, Libing
Li-Bing Zhang, Dr. nat., Curator, Missouri Botanical Garden
www.mobot.org/Li-Bing Zhang
<https://www.missouribotanicalgarden.org/plant-science/plant-science/research-staff/article/2628/zhang-li-bing>

**Comment by MichaelKessler979** [comment]
I don't know if this will help, but I agree that splitting Selaginella will likely happen and makes sense to some degree, there is a lot of variability in the genus and those working with it often use the subgenera to bring order into the chaos. So why not elevate these to genera? On the other hand, I feel that splitting the genus into 19 genera, some of which are ill-defined and have very few species may be too big a change at present. So, why not for now recognize the 7 subgenera as genera? If a proposal is made for recognition of the 19 genera, it may make sense to make an additional proposal for recognition of 7 genera. I don't know if this is possible, but maybe the two options can be presented in a single proposal?

**Reply by joelnitta** [reply]
It is preferred to write the proposal as single taxonomic change (or set of closely related changes, all stemming from the same concept). If you are in favor of a 7 genus system, I think you should submit that as a proposal.
